core/templates/dev/head/pages/exploration_editor/RouterServices.js

```diff
@@ -85,6 +85,17 @@ oppia.factory('routerService', [
       }
     });
 
+    // Select2 dropdowns does not automatically get populated
```

Comment:
Can we find a way to instead do this only in the settings tab controller, when it is "loaded"? E.g. maybe put an ng-if around the dropdown fields that causes them to show only when the settings tab is active, or something like that?

RouterServices should not know anything about select2 dropdowns on individual places.